clarity_ext/service/dilution/service.py

```diff
@@ -71,7 +71,7 @@ def evaluate(self, pairs):
         """Refreshes all calculations for all registered robots and runs registered handlers and validators."""
         self.pairs = pairs
         self.transfer_batches_by_robot = dict()
-        for robot_settings in self.robot_settings_by_name.values():
+        for robot_settings in list(self.robot_settings_by_name.values()):
```

Comment:
Here is a gain an example of where `2to3` isn't smart enough :) Since we're just iterating we don't need to add it to a list.

Reply:
:+1: